### ISSUE TITLE
Support installing CUDA Quantum in a local directory

### ISSUE THREAD

**Issue by bettinaheim** [NVIDIA/cuda-quantum#1075]
CUDA Quantum currently finds some of its dependencies (cuQuantum, cuTensor) via the rpaths configured during build.
These paths are absolute, making the installation non-relocatable. As a consequence, installing pre-built binaries requires admin privileges to install the dependencies in global locations. 

It would be nice if it was possible to install CUDA Quantum in a user directory without requiring building CUDA Quantum from source. 
Specifically, the cuda_quantum_installer should take the installation location as argument. It would also be nice if the installer checked 
the system pre-requisites before proceeding with the installation.

**Comment by diehlpk** [comment]
I agree. I wanted to explore the software but hesitated to ask our admin for a cluster wide installation.

**Comment by diehlpk** [comment]
@mmvandieren that is the issue we chatted about.

**Comment by diehlpk** [comment]
Any update on that issue? I would like to run CUDA-Q on a cluster with no container support.
